config/routes.rb

```diff
@@ -3,6 +3,8 @@
   get "/about", to: "static_pages#about"
   get "/contact", to: "static_pages#contact"
   get "/login", to: "static_pages#login"
-  get "/signup",  to: "users#new"
+  get "/signup", to: "users#new"
+  post "/signup", to: "users#create"
   root "static_pages#home"
+  resources :users
```

Comment:
resources có tác dụng gì

Reply:
resources có tác dụng tạo ra 7 routes cơ bản là 
+ /users : action index, show all users
+ /users/1 : action show, show 1 user
+ /user/new : action new, show page to make a new user
+ /users : POST - create new user, create a new user
+ users/1/edit: action edit , show page to edit a user
+ users/1 : PATCH - action update, update a user
+ users/1: DELETE - action delete 1 user, delete user